structures/auction.js

```diff
@@ -43,12 +43,12 @@ exports.auction = function (config = null) {
       bestbid: s.optional(Uuid),
       getitnow: s.optional(s.number()),
       winningbid: s.optional(Uuid),
-      puPlace: s.dynamic((v, p) => { return v.length === 5 ? Place : PlaceTZ }),
+      puPlace: s.dynamic((v, p) => { return v && v.length === 5 ? Place : PlaceTZ }),
```

Comment:
Mumm sound weird here... Can we use `array` before `dynamic` ? That will make puPlace and dePlace mandatory.

Reply:
PuPlace and dePlace are mandatory. Creation failed with 400 if not provided.
Test auctionF6 is testing this scenario. Unless I fail. 😁